pinot-spi/src/main/java/org/apache/pinot/spi/config/table/UpsertConfig.java

```diff
@@ -57,12 +57,18 @@ public enum Strategy {
   @JsonPropertyDescription("Whether to use snapshot for fast upsert metadata recovery")
   private boolean _enableSnapshot;
 
+  @JsonPropertyDescription("Whether to use TTL to reduce upsert metadata memory footprint")
+  private boolean _enableTTL;
```

Comment:
Suggest removing this. Providing `_upsertTTLConfig` implicitly indicate that TTL is enabled